spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+feature 'Comments' do
+  # As a user
+  # So that I can share my thoughts on a picture
+  # I would like to write a comment on a picture
+
+  context 'when signed in' do
+  let!(:user){User.create(name: 'Betty Draper', email: 'betty@drapers.com',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.